nengo_viz/static/viz_modal.js

```diff
@@ -61,6 +61,20 @@ VIZ.Modal.prototype.text_body = function(text, type) {
     $p.append(document.createTextNode(text));
 }
 
+VIZ.Modal.prototype.help_body = function() {
+    this.clear_body();
+
+    $('.modal-dialog').addClass('modal-sm');
```

Comment:
This class will persist on other modals. Can you add something like `$('.modal-dialog').removeClass('modal-sm')` to `VIZ.Modal.prototype.clear_body`? Also, it would be better to be specific (i.e., `this.$div.find('.modal-dialog')`, or capture this as `this.$dialog = this.$div.find('.modal-dialog').first()` in the constructor).